### DISCUSSION TITLE
Dependencies and start up

### DISCUSSION THREAD

**Discussion by Jenfer93** [Fermain/-mollify#13] [Q&A]
Hi guys! 

Want to setup my end of the project correctly from the start, and just wanted to ask if there is possible for a overview of the installments(dependencies) that needs to get done from the get-go so we have the same ones? 
Or am I overthinking this (I might have a tendency to do that)

**Comment by Fermain** [comment]
In a lerna monorepo, multiple packages exist within the same repository, and each package may have its own set of dependencies. This makes the task more complicated than usual as we have the option to install a dep for a single package, or globally for the entire project.

For example, it is likely that all of the current packages (lms, flow, molly, cli) will require `marked` as a dependency - as this is used for parsing markdown.

This means we need to think of 4 types of dependency instead of 2:
1. global dependency
2. global dev dependency
3. local dependency
4. local dev dependency

**Reply by Jenfer93** [reply]
Ok, so I'm guessing stuff like mdsvex should be global dependency? 
I'm not quite sure I fully understand which we are supposed to use yet and which we've just loosely discussed! 

Should we create a overview over the installations that are needed where and what we have agreed to use, and a discussion to discuss the ones we haven't agreed on?

**Reply by Fermain** [reply]
For now lets just install packages per project, then we can consolidate global packages later on

**Reply by Jenfer93** [reply]
sure!
